.github/dependabot.yml

```diff
@@ -0,0 +1,24 @@
+#
+# Copyright (c) 2021, 2022 Contributors to the Eclipse Foundation
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Public License v. 2.0 which is available at
+# http://www.eclipse.org/legal/epl-2.0,
+# or the Eclipse Distribution License v. 1.0 which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
```

Comment:
this project uses `EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0`, the text is ie [here](https://github.com/jakartaee/common-annotations-api/blob/master/api/pom.xml)